test/models/user_test.rb

```diff
@@ -0,0 +1,7 @@
+require "test_helper"
+
+class UserTest < ActiveSupport::TestCase
+  # test "the truth" do
+  #   assert true
+  # end
```

Comment:
Why are the tests commented out here and in all of the other files? We should either uncomment and let them run or delete them.